### DISCUSSION TITLE
Sorting by date

### DISCUSSION THREAD

**Discussion by JhonnyAxe** [dailydotdev/daily#810] [Feature Requests]
Could you please add sorting by date feature? It's pretty inconvenient to read...

**Comment by sshanzel** [comment]
Hi! Have you tried checking this part of our docs page (link below)? I think this might help. Let me know if it didn't work for you 🙏 

https://docs.daily.dev/docs/customize-your-feed/preferences#show-feed-sorting-menu

**Reply by michabbb** [reply]
when you search for something and see the result, **THERE** it would be important to sort as well. i don't want to sort out stuff that is 6 months old when looking into specific topics... thanks !! 🙏 

for instance: if you search for "laravel" I only see stuff from 2022 😕 
so it would be nice to have a default-sort that is saved within my profile.

this lets me think: it would be even better to have an option I can use **BEFORE** I search, for example:

- not older than 1 day
- not older than 1 week
- not older than 1 month
- not older than 1 year

that would be perfect 😁

**Reply by sshanzel** [reply]
Hi! Thank you for the detailed response! I believe there is an open discussion about this topic. I think we can keep the conversation there. Appreciate the feedback though! 🥇 

https://github.com/dailydotdev/daily/discussions/294#discussioncomment-4366846

**Comment by JhonnyAxe** [comment]
Got it! Thanks С уважением, Павел89040429532On 6 Feb 2023, at 18:03, Lee Hansel Solevilla ***@***.***> wrote:﻿
Hi! Have you tried checking this part of our docs page (link below)? I think this might help. Let me know if it didn't work for you 🙏
https://docs.daily.dev/docs/customize-your-feed/preferences

**Reply by sshanzel** [reply]
Happy to know it helped! 🚢
